classes/controllers/FrmSimpleBlocksController.php

```diff
@@ -0,0 +1,207 @@
+
+		return $show_counts;
+	}
+
+	/**
+	 * Get the specified meta value for the specified post type
+	 *
+	 * @param string $key
+	 * @param string $post_type
+	 *
+	 * @return array|object|void|null
+	 */
+	private static function get_meta_values( $key = '', $post_type = 'frm_display' ) {
+
+		global $wpdb;
+
+		if ( empty( $key ) ) {
+			return;
+		}
+
+		$results = $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT pm.post_id, pm.meta_value, pm.meta_key FROM {$wpdb->postmeta} pm LEFT JOIN {$wpdb->posts} p ON p.ID = pm.post_id WHERE ( pm.meta_key = %s ) AND p.post_type = %s",
+				$key,
+				$post_type
+			),
+			OBJECT_K
+		);
+
+		return $results;
+	}
+
+	/**
+	 * Get the options for the site's Views
+	 *
+	 * @return array|object|void|null
+	 */
+	private static function get_frm_options_for_views() {
+
+		$views_options = self::get_meta_values( 'frm_options', 'frm_display' );
+
+		foreach ( $views_options as $key => $value ) {
+			$views_options[ $key ]->meta_value = unserialize( $value->meta_value );
+		}
+
+		return $views_options;
+	}
+
+	/**
+	 * Registers simple form and View blocks
+	 *
+	 */
+	public static function register_guten_blocks() {
+		if ( ! is_callable( 'register_block_type' ) ) {
+			return;
+		}
+
+		register_block_type(
+			'formidable/simple-form',
+			array(
+				'attributes'      => array(
+					'form_id'     => array(
+						'type' => 'string',
+					),
+					'title'       => array(
+						'type' => 'string',
+					),
+					'description' => array(
+						'type' => 'string',
+					),
+					'minimize'    => array(
+						'type' => 'string',
+					),
+				),
+				'editor_script'   => 'formidable_simple-block-js',
+				'render_callback' => array( 'FrmSimpleBlocksController', 'simple_form_render' ),
+			)
+		);
+
+		register_block_type(
+			'formidable/simple-view',
+			array(
+				'attributes'      => array(
+					'view_id'           => array(
+						'type' => 'string',
+					),
+					'filter'            => array(
+						'type' => 'string',
+					),
+					'limit'             => array(
+						'type' => 'string',
+					),
+					'use_default_limit' => array(
+						'type'    => 'boolean',
+						'default' => false,
+					),
+				),
+				'editor_script'   => 'formidable_simple-block-js',
+				'render_callback' => array( 'FrmSimpleBlocksController', 'simple_view_render' ),
+			)
+		);
+	}
+
+	/**
+	 * Renders a form given the specified attributes.
+	 *
+	 * @param $attributes
+	 *
+	 * @return string
+	 */
+	public static function simple_form_render( $attributes ) {
+		$params = '';
+		$params .= self::create_attribute_text( 'id', $attributes['form_id'] );
+		$params .= self::create_attribute_text( 'title', $attributes['title'] );
+		$params .= self::create_attribute_text( 'description', $attributes['description'] );
+		$params .= self::create_attribute_text( 'minimize', $attributes['minimize'] );
+
+		return do_shortcode( '[formidable' . $params . ']' );
```

Comment:
Looks like the stylesheet is missing.
```suggestion
		$form = do_shortcode( '[formidable' . $params . ']' );
		ob_start();
		wp_print_styles( 'formidable' );
		$form .= ob_get_contents();
		ob_end_clean();
		return $form;
```

Reply:
Thank you, thank you, thank you!  We desperately needed this.  I didn't know how to do it.  Once again, you've saved the day!